src/zoid/buttons/util.js

```diff
@@ -357,3 +358,45 @@ export function getButtonSize(
     }
   }
 }
+
+// eslint-disable-next-line promise/no-native, no-restricted-globals
+export async function getModal(): string | Promise<Object> | void {
+  const modalBundleUrl = {
+    local:
+      "https://www.paypalobjects.com/upstream/bizcomponents/stage/modal.js",
+    test: "https://www.paypalobjects.com/upstream/bizcomponents/stage/modal.js",
+    stage:
+      "https://www.paypalobjects.com/upstream/bizcomponents/stage/modal.js",
+    sandbox:
+      "https://www.paypalobjects.com/upstream/bizcomponents/sandbox/modal.js",
+    production:
+      "https://www.paypalobjects.com/upstream/bizcomponents/js/modal.js",
+  };
```

Comment:
Instead of hardcoding the URL string multiple times increasing the bundle size, could we just fill in the `/js/` / `/sandbox/` / `/stage/` piece as is appropriate based on the environment?